archey/logos/moevalent.py

```diff
@@ -0,0 +1,23 @@
+"""Moevalent logo"""
+
+from archey.colors import Colors
+
+COLORS = [Colors.RED_BRIGHT, Colors.MAGENTA_BRIGHT, Colors.WHITE_BRIGHT]
+
+# pylint: disable=line-too-long
```

Comment:
```suggestion
```

I guess this is not required here 😉

Reply:
I corrected the relevant part, so please check it.